docs/ReleaseHistory.md

```diff
@@ -11,6 +12,11 @@
 - FNS => Flase negative reduction in static analysis.
 
 # UNRELEASED
+- RUL: Add `SEC101/051.LooseSasSecret` detection.
+- RUL: Add `SEC101/528.GenericJwt` detection.
+- BRK: Rename `WellknownPatterns.HighConfidenceSecurityModels` to `WellknownPatterns.PreciselyClassifiedSecurityKeys`.
+- BRK: Rename `WellknownPatterns.LowConfidencePotentialSecurityKeys` to `WellknownPatterns.UnclassifiedPotentialSecurityKeys`.
```

Comment:
The `HighConfidence` and `LowConfidence` nomenclature was better. It conveys some information about the detection capability. For `Unclassified`, it's less clear---why is it unclassified?

This is a not a blocking comment.

Reply:
I don't agree. In fact, the literal accuracy/confidence of these checks varies and in some cases, a rule that doesn't actually classify a finding may be fairly high confidence. In practice, our generic JWT rule doesn't fire many false positives for example.

So what I've done is separate rules by those that attempt to specifically classify a finding (this is an event hub key) vs. those that don't (this is an unclassified JWT).

These designations say something about volume of finds and the literal utility of findings, which is why I separated them.

Categorization is hard, so I'm definitely interested in the ongoing conversation here. What does 'High' vs. 'Low' confidence mean to you? What criterion should we use to decide where a rule would be placed given these two buckets? This is where I was struggling.

Reply:
I'd prefer that if a rule fires some false positives, then we could put it in Low confidence. If no false positive feedback received or identified, then we can leave it in High confidence. So basically all rules are high confidence, we just need a new file for low confidence when we find false positives from Classified patterns or Unclassified patterns